### DISCUSSION TITLE
iOS 16 Home App Update - Dryer Tile Gone

### DISCUSSION THREAD

**Discussion by ozTahu** [nVuln/homebridge-lg-thinq#185] [Q&A]
Hi there all!

Not sure if anyone has come across this yet...

It seems the Dryer tile that used to display the time left, has been removed in iOS 16 update of the home app.  

It has been replaced with by a simple icon of a tap named "Valve" with "Running" underneath, all at the top of the screen.

My question is are we able to change it back to a tile so we can have the time countdown back?

Clicking on it does not seem to offer a choice of how its displayed.

Any advice would be appreciated!

![Screenshot](https://user-images.githubusercontent.com/91027943/191366051-df5c58e2-1c9f-4d2f-bc01-e90e60575dce.jpg)

**Comment by nVuln** [comment]
not sure what happened, my home app still showing like this

![IMG_0353](https://user-images.githubusercontent.com/54855446/191390561-ff333777-ac6e-45bd-a396-6a15d55050b1.PNG)

**Reply by ozTahu** [reply]
I went into the plugin settings and checked the "enable program finished trigger?"  with the below result.  

Could it be that the dryer is running "thiq1"?

![Screenshot02](https://user-images.githubusercontent.com/91027943/191392212-64d39e74-86c2-46bd-8f68-73bf3aa16ed2.jpg)

**Comment by tryckman** [comment]
I'm experiencing the same behavior with iOS 16.
